worker/dbaccessor/worker.go

```diff
@@ -416,10 +417,22 @@ func (w *dbWorker) initialiseDqlite(options ...app.Option) error {
 		return errors.Trace(err)
 	}
 
-	ctx, cancel := w.scopedContext()
-	defer cancel()
+	ctx, pCancel := w.scopedContext()
+	defer pCancel()
+	ctx, cCancel := context.WithTimeout(ctx, time.Minute)
+	defer cCancel()
 
 	if err := w.dbApp.Ready(ctx); err != nil {
+		if err == context.DeadlineExceeded {
+			// We don't know whether we were cancelled by tomb or by timeout.
```

Comment:
See: https://pkg.go.dev/context#WithCancelCause

Reply:
That doesn't help use here, because neither the catacomb nor the time-out will cancel with a cause we can access this way.